packages/geo/index.d.ts

```diff
@@ -116,6 +117,7 @@ declare module '@nivo/geo' {
         onMouseMove?: ChoroplethEventHandler
         onMouseLeave?: ChoroplethEventHandler
         onClick?: ChoroplethEventHandler
+        legends: Array<{ dataFrom: 'indexes' | 'keys' } & LegendProps>
```

Comment:
This package doesn't support `dataFrom`, the type should be `legends: LegendProps[]`